### DISCUSSION TITLE
Tabs being rendered one below the other and not side by side

### DISCUSSION THREAD

**Discussion by hudsondiegof** [squidfunk/mkdocs-material#6171] [Q&A] [answered]
When I try to use tabs they are rendered one below the other and not side by side.

```
# Test

=== "Tab 1"
- test1
- test2
- test3

=== "Tab 2"
- test1
- test2
- test3
```

Output:
![image](https://github.com/squidfunk/mkdocs-material/assets/83621099/2ad9a5ec-a589-4ade-a708-4ff5c313b1fa)

mkdocs.yml

![image](https://github.com/squidfunk/mkdocs-material/assets/83621099/5c642daf-e1b0-4b26-87b1-ad96997a136f)

**Comment by facelessuser** [comment, marked as answer]
Tabs, like admonitions, require the content to be indented 4 spaces under them.

**Comment by hudsondiegof** [comment]
thank you very much
